posts/2020-06-03-asynchronous-programming-made-resilient-with-mpft.adoc

```diff
@@ -86,21 +83,21 @@ public CompletionStage<String> clientDemo() {
 }
 ----
 
-If we call the `clientDemo()` method, without any annotations, it works as we expect. We call it, receive a `CompletionStage<String>` (named `response` in the example), and then we can add an action to take when the `CompletionStage` completes:
+If we call the `clientDemo()` method without any annotations, it works as we expect. We call it, receive a `CompletionStage<String>` (named `response` in the example), and then we can add an action to take when the `CompletionStage` completes:
 
 [source,java]
 ----
 response.thenAccept(System.out::println);
 --> responseText
 ----
 
-However, what happens if we want to use the `@Retry` annotation to retry any failed requests with our method? The answer is that the method will not be retried.
+However, if we want to use the `@Retry` annotation to retry any failed requests, the method isn't retried.
```

Comment:
Not sure this sentence really makes sense. The start of the method talks about wanting to use the retry annotation, but the end of the sentence just states that the method isn't retried, as if we really had added the retry annotation, rather than just thinking about it.

How about this instead:
```suggestion
If we wanted to retry any failed requests, we might add the `@Retry` annotation to the method, but surprisingly this wouldn't work!
```